src/components/Countries/SubComponent/DropdownFilter/DropdownFilter.tsx

```diff
@@ -0,0 +1,43 @@
+'use client'
+import Link from 'next/link'
+import Style from './DropdownFilter.module.scss'
+import DropdownSelect from '@/components/common/form-control/DropdownSelect'
+
+export default function DropdownFilter({
+  searchParams,
+}: {
+  searchParams: {
+    region: string
+    search: string
+  }
+}) {
+  const { region, search } = searchParams
+  const regionItems = ['Africa', 'America', 'Asia', 'Europe', 'Oceania']
```

Comment:
Extract the regionItems to a constant outside of the component. This way, the list isn't recreated every time the component re-renders, maybe just above ur export default since its only used in this component. otherwise if used elsewhere then it make sense to have constants file on its own

Reply:
Thankyou for the review. I will do that.